src/core/worker.js

```diff
@@ -724,8 +724,12 @@ var WorkerMessageHandler = {
     );
 
     handler.on('GetAnnotations', function wphSetupGetAnnotations(data) {
-      return pdfManager.getPage(data.pageIndex).then(function(page) {
-        return pdfManager.ensure(page, 'getAnnotationsData', [data.intent]);
+      var pageIndex = data.pageIndex;
```

Comment:
No need for this intermediate variable; just keep it like before since it should fit on one line.